app/src/main/java/org/simple/clinic/patientcontact/PatientContactUi.kt

```diff
@@ -4,6 +4,13 @@ import org.simple.clinic.patient.Gender
 
 interface PatientContactUi {
   fun renderPatientDetails(name: String, gender: Gender, age: Int, phoneNumber: String)
+
   fun showCallResultSection()
   fun hideCallResultSection()
+
+  fun showSecureCallButton()
+  fun hideSecureCallButton()
+
+  fun showSecureCallMessage()
+  fun hideSecureCallMessage()
```

Comment:
A doubt. Is the toggling logic for secure call button and message different?

Reply:
It is not, I can unify them.